web/src/style-helpers.ts

```diff
@@ -13,6 +13,7 @@ export enum Color {
 
   offWhite = "#eef1f1",
   grayLightest = "#93a1a1", // Solarized base1 (darkest content tone)
+  grayLighter = "#2D4D55",
```

Comment:
We're woefully limited in grays specified in style-helpers *vs* grays in use in Figma. I've worked around this in the past by using `Color.white` or `Color.black` with alpha transparency, but feels _mildly_ computationally expensive when used in UI elements that can proliferate. Also, this alpha method generates random grays outside our official color scale.

FWIW: I'm irritated at my past self for coming up with a Color naming scheme that isn't very friendly to expansion! Next time I have a chance, I'm going to use a [number-based color scale](https://dev.to/ynab/a-semantic-color-system-the-theory-hk7)!

Reply:
ya, after we throw away the old UI i want to go back and align these with the color scale we have in figma